Winapp2.ini

```diff
@@ -1897,17 +1865,12 @@ FileKey2=%AppData%\aignes\website-watcher\config\log|*.*
 FileKey3=%AppData%\aignes\WebSite-Watcher\config\settings|*.cfg_bak*
 FileKey4=%AppData%\aignes\website-watcher\config\temp|*.*
 
-[AIM Cache*]
-LangSecRef=3022
-DetectFile=%LocalAppData%\AOL\AIM
-Default=False
-FileKey1=%LocalAppData%\AOL\AIM\cache|*.*|REMOVESELF
-
-[AIM Logs*]
+[AIM*]
 LangSecRef=3022
 DetectFile=%LocalAppData%\AOL\AIM
 Default=False
 FileKey1=%LocalAppData%\AOL\AIM|*.log|RECURSE
+FileKey2=%LocalAppData%\AOL\AIM\cache|*.*|REMOVESELF
```

Comment:
We should leave cache files in there own entry as these files are meant to speed up a program and should only be removed if you need disk space. Also, most programs just regenerate cache files, which might make too many writes on SSDs.